docs/fa/examples.md

```diff
@@ -0,0 +1,97 @@
+# Examples
+
+[FA Class](class-fa.md)  
+[NFA Class](class-nfa.md)  
+[DFA Class](class-dfa.md)  
+[Table of Contents](../README.md)
+
+On this page, we give some short examples with discussion for the finite automata classes and methods.
+
+## Reading basic input
+
+The following code snippet creates a `DFA` and prints whether it accepts or rejects user input.
+
+```python
+from automata.fa.dfa import DFA
+
+# DFA which matches all binary strings ending in an odd number of '1's
+my_dfa = DFA(
+    states={'q0', 'q1', 'q2'},
+    input_symbols={'0', '1'},
+    transitions={
+        'q0': {'0': 'q0', '1': 'q1'},
+        'q1': {'0': 'q0', '1': 'q2'},
+        'q2': {'0': 'q2', '1': 'q1'}
+    },
+    initial_state='q0',
+    final_states={'q1'}
+)
+
+try:
+    while True:
+        if my_dfa.accepts_input(input('Please enter your input: ')):
+            print('Accepted')
+        else:
+            print('Rejected')
+except KeyboardInterrupt:
+    print('')
+```
+
+## Subset for NFAs
+
+The `NFA` does not have a built-in method for checking whether it is a subset of another `NFA`. However, this can be done using existing methods.
+
+```python
+# In the following, we have nfa1 and nfa2 and want to determine whether nfa1 is a subset of nfa2.
```

Comment:
@eliotwrobson I hate to be nitpicky, but are you able to reflow this comment onto two lines? I like to have all my lines fit within 80 characters, particular in the documentation.

Reply:
No worries! This is a pretty easy thing to do.

Reply:
Done here https://github.com/caleb531/automata/pull/146/commits/2ce6816e437d8ca17b4e095ab4922e18eb4421f7